content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -42,74 +136,70 @@ API connections, and its
 
 ⚠️ It should be safe to share **client ID** and **client secret** among your
 team. These credentials are only used to generate the
-[authorization](#authorization) URL DVC will later prompt to visit in order to
+[authorization](#authorization) URL you'll need to visit later in order to
 connect to the Google Drive.
 
-> Note that Google Drive API has usage limits/quotas per credentials in a
-> _project_ (which you can review in the
-> [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent)).
-> Please keep this in mind when sharing them, or you may
-> [exceed the limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#resolve_a_403_error_usage_limit_exceeded).
-
-## URL format for Google Drive remotes
-
-A GDrive DVC remote URLs is needed for the `dvc remote add` command. It can be
-constructed with a _base_, and an **optional** _path_ to a folder i.e.
-`gdrive://<base>/path/to/folder`. The base can be one of:
-
-1. `root` - indicates your topmost Google Drive directory.
+Finally, use the `dvc remote modify` command to set the credentials (for each
+GDrive remote), for example:
 
 ```dvc
-$ dvc remote add -d mygdroot gdrive://root
-$ dvc remote add mygdfolder gdrive://root/path/to/folder
+$ dvc remote add gdfolder gdrive://0AIac4JZqHhKmUk9PDA
```

Comment:
probably we should not mention this command here - better to mention basic workflow and specify additional commands only

Reply:
Removed.